tasks-by-users/anask601/01-js-portfolio/index.html

```diff
@@ -0,0 +1,923 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <!--------------------------------------unicons----------------------------------->
+        <link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.0/css/line.css"/>
+        <!--------------------------------------swiper css----------------------------------->
+        <link rel="stylesheet" href="assets/css/swiper-bundle.min.css" />
+        <!--------------------------------------css----------------------------------->
+        <link rel="stylesheet" type="text/css" href="assets/css/styles.css" />
+        <title>Portfolio</title>
+  </head>
+  <body>
+    <!----------------------------------------- HEADER ---------------------------------------------->
+      <header class="header" id="header">
```

Comment:
For tags like header,footer etc you dont need to add class or id, you could directly access it with css selector like 

header {
}

footer {
}

Reply:
okay